runtime/ftplugin/rsc.vim

```diff
@@ -0,0 +1,6 @@
+" Vim filetype plugin file
+" Language:     RouterOS scripts
+" Maintainer:   zainin
+" Last Change:  2017-07-26
+
+au BufRead,BufNewFile *.rsc set filetype=rsc
```

Comment:
This should go into filetype.vim, not into a filetype plugin.

Reply:
My bad, fixed.